specification/schemas/ShipmentSurcharge.json

```diff
@@ -0,0 +1,82 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalShipmentSurcharge"
+    },
+    {
+      "required": [
+        "attributes",
+        "relationships"
+      ],
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "amount",
+            "currency"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "name": {
+              "type": "string",
+              "example": "Re-labeling surcharge"
+            },
+            "description": {
+              "type": "string",
+              "example": "A carrier-specific (carrier_code) re-labeling surcharge"
+            },
+            "amount": {
+              "type": "integer",
+              "example": 523,
+              "description": "Surcharge amount in cents/pence."
+            },
+            "currency": {
+              "$ref": "#/components/schemas/Currency"
+            },
+            "created_at": {
+              "$ref": "#/components/schemas/Timestamp"
+            },
+            "updated_at": {
+              "$ref": "#/components/schemas/Timestamp"
+            }
+          }
+        },
+        "relationships": {
+          "type": "object",
+          "required": [
+            "shipment"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "shipment": {
+              "allOf": [
+                {
+                  "$ref": "#/components/schemas/ShipmentRelationship"
+                },
+                {
+                  "readOnly": false
```

Comment:
Is the ShipmentRelationship "readOnly"?

Reply:
If readOnly is set to `true` the example request will be empty in the specs page.

Reply:
No I understand that, I was just wondering if the ShipmentRelationship schema includes `"readOnly": true`. Since you're trying to "change" it to false here.

Reply:
It is readOnly: true